### PR TITLE
x86_cpu_model: update some cases' steps

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1966829

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@vivianQizhu @huangyum Please help to review this, thanks.

And Yumei, do we need to check the other flags inside guest?

**Comment by huangyum** [comment]
> And Yumei, do we need to check the other flags inside guest?
I think it's ok to not check flags in rhel6 and rhel7 guest. 
The commit summary is not reflecting the code change. Do you want to update only EPYC-Rome case or other cases as well? What's the reason to only update some of the cases?

**Comment by nanliu-r** [comment]
> > And Yumei, do we need to check the other flags inside guest?
> 
> I think it's ok to not check flags in rhel6 and rhel7 guest.
> The commit summary is not reflecting the code change. Do you want to update only EPYC-Rome case or other cases as well? What's the reason to only update some of the cases?

Sorry for the later reply, I have updated the summary to cover other cases since we can reproduce this issue with the other model which have the flag "wbnoinvd".

**Comment by huangyum** [comment]
The commit summary is not updated. Suggest to explain what and why the changes in commit message.

File: qemu/tests/cfg/x86_cpu_model.cfg [hunk 1]
```diff
@@ -29,6 +29,8 @@
             only HostCpuVendor.amd
             flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
             model_pattern = "AMD EPYC-Rome Processor%s"
+            RHEL.6,RHEL.7:
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_model.cfg:32]: Need add space after ','.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_model.cfg]: > Need add space after ','.

@huangyum update and thanks.

**Comment by huangyum** [comment]
> The commit summary is not updated. Suggest to explain what and why the changes in commit message.

@nanliu-r Missed this comment?

**Comment by nanliu-r** [comment]
> > The commit summary is not updated. Suggest to explain what and why the changes in commit message.
> 
> @nanliu-r Missed this comment?

sorry, I have updated the summary, thanks.

**Comment by nanliu-r** [comment]
@huangyum please help to review again, thanks.

**Review by huangyum — APPROVED** [review]
ACK

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**